docker-compose.yml

```
@@ -52,7 +36,7 @@
         condition: service_healthy
     restart: unless-stopped
     healthcheck:
-      test: ['CMD', 'wget', '--spider', '-q', 'http://localhost:3000/api/health']
+      test: [ 'CMD', 'wget', '--spider', '-q', 'http://localhost:3000/api/health' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)